sdk/lib/atl/atltrace.h

```diff
@@ -17,6 +17,8 @@
 extern "C"
 {
 // FIXME: Enabling _DEBUG at top level causes assertion failures... CORE-17505
+#undef _CrtDbgReport
+#undef _CrtDbgReportW
 int __cdecl _CrtDbgReport(int reportType, const char *filename, int linenumber, const char *moduleName, const char *format, ...);
```

Comment:
#ifdef _DEBUG
EXTERN_C int __cdecl _CrtDbgReport(int reportType, const char *filename, int linenumber, const char *moduleName, const char *format, ...);
...
#endif

#ifdef _DEBUG
... (`_CrtDbgReport` usage) ...
#endif

Reply:
#include <crtdbg.h>

Reply:
Follow the One Definition Rule:
https://en.m.wikipedia.org/wiki/One_Definition_Rule

Reply:
`_CrtDbgReport` should be declared in `<crtdbg.h>`. Not here.